### ISSUE TITLE
[Question] Autogenerated description for IAM Roles created by CDK

### ISSUE THREAD

**Issue by robertd** [aws/aws-cdk#6235]
## :question: General Issue

Now that #6179 has landed, it would be really nice if we can have autogenerated descriptions for roles created automatically by CDK. This would be really helpful, especially if you're pushing CDK within your team and there are 100's of roles, and they don't necessarily like add IAM Role description field (se screenshot below). 

![image](https://user-images.githubusercontent.com/31543/74267426-4220c900-4cc3-11ea-91b5-c1b930534420.png)

### The Question

Where would this logic go? In `aws-iam` package, or each package should populate `description` field separately. In any case tons of tests would have to be fixed all across the board.

I would just like to start a discussion and see where it goes.

**Comment by skinny85** [comment]
I think this is a great idea :)

**Comment by rix0rrr** [comment]
Yeah, that's a good idea. It should go into every package that individually creates a role (should tell you that it was automatically generated by the CDK and what the role is used for).

**Comment by robertd** [comment]
I'm planning on reviving #6393 when time permits.
